frictionless/plugins/pandas/parser.py

```diff
@@ -167,6 +167,21 @@ def write_row_stream(self, resource):
 
         # Create/set dataframe
         dataframe = pd.DataFrame(data_rows, index=index, columns=columns)
```

Comment:
Perhaps it would be best performance wise to instantiate the DataFrame with the correct `dtype` instead of converting it afterwards.

The problem is that [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html#pandas-dataframe) accepts only a single `dtype`. So we would need to do some prep work, like creating a `Series` for each column or something like that.